packages/@okta/vuepress-theme-prose/assets/css/components/_top-bar.scss

```diff
@@ -1,92 +1,263 @@
 $gray-900: #2f3f4a;
-.top-bar {
-  display: block;
-  background-color: cv('brand');
-  padding: 0rem;
-  z-index: 20;
-  border-bottom: none;
 
-  &--search {
-    &--mobile {
-      @include media(">=tablet") {
-        display: none;
-      }
+.top-bar-nav{
+  display: flex;
+  flex-direction: row;
+  padding: 30px;
```

Comment:
top-level vertical padding is doing a good job at keeping header height dependent on its contents - any objections on keeping this structure?(i.e. aligning header contents to 35px height)

Reply:
I will disagree here. I would vote to stick to this solution due to simpleness of this approach. I've had a hard times trying to understand sequence and positioning of elements in /blog header and I want to avoid that situation in future. 

Anyway, I've deleted fixed height and adjusted paddings to make CSS more clean.